environ/ci

```diff
@@ -1,5 +1,15 @@
 #! /usr/bin/env bash
 
+SERVICE_NAME=identity-service
+
 ./gradlew init
 ./gradlew build
-./gradlew test
\ No newline at end of file
+./gradlew test
+
+docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
+docker pull cshr/${SERVICE_NAME} || true
+
+docker build -t cshr/${SERVICE_NAME}:latest .
+
```

Comment:
INFO - Our new versioning (till get get the CI sorted
```
TAG =${TRAVIS_TAG:-$TRAVIS_COMMIT}
docker tag cshr/${SERVICE_NAME}:latest cshr/${SERVICE_NAME}:${TAG}
```

Reply:
👍 sorted